src/backend/gl/src/lib.rs

```diff
@@ -197,6 +334,18 @@ impl PhysicalDevice {
         }
         let name = info.platform_name.renderer.into();
 
+        // Because some drivers don't actually support this stuff, but report
+        // that they do, we got to disable their private cap.
+        {
+            if !gl.GenVertexArrays.is_loaded {
```

Comment:
Can you call `gl.GenVertexArrays.is_loaded()` here instead?

Reply:
Haha, I didn't actually see that function when scrolling through the generated file. Thanks.

Reply:
Yeah, might be more of a bug with my documentation, if anything!